tools/js-sdk-release-tools/src/xlc/codeUpdate/updateTspLocation.ts

```diff
@@ -0,0 +1,10 @@
+import * as fs from "fs";
```

Comment:
please merge this file with the one to update user agent, make them into: `tools/js-sdk-release-tools/src/xlc/codeUpdate/generatedSdkUpdater.ts`